features/manage-position/Create.tsx

```diff
@@ -37,14 +37,14 @@ const Create = () => {
     decimals: collDec,
     allowance: collAllowance,
     setMaxAllowance,
-    balance
+    balance,
```

Comment:
@chrismaree @adrianmcli I've noticed that there are trailing commas like this. Any idea why this is "good practice" lol?